src/check-nsfw.js

```diff
@@ -86,9 +86,9 @@ async function checkNSFW(message, urls) {
 				return (previous.probability > current.probability) ? previous : current;
 			});
 
+			if (classification.probability < 0.8) return;  // if low confidence then do nothing
```

Comment:
Just remembered that this check is _also_ in a loop (the bot checks _ALL_ urls it finds in a message) so this should also be `continue`. My bad

Reply:
Oops yeah completely missed that, my bad, I should've checked :/
Anyways, now it's fixed!